### PR TITLE
Feature/drone return home

### PR DESCRIPTION
**Pull request by AlexisTabin**
Very simple functionality that is able to call the drone back to where it has take off

### CODE REVIEW

File: app/src/androidTest/java/ch/epfl/sdp/MapActivityTest.kt [hunk 1]
```diff
@@ -68,19 +69,21 @@ class MapActivityTest {
...
-                    LatLng(8.544867, 47.397426),
-                    LatLng(8.543067, 47.397026)
+                    LatLng(47.398979,  8.543434),
+                    LatLng(47.398279, 8.543434),
+                    LatLng(47.397426, 8.544867),
+                    LatLng(47.397026, 8.543067)
             ).forEach { latLng -> mActivityRule.activity.onMapClicked(latLng) }
         }
         onView(withId(R.id.start_mission_button)).perform(click())
+        Thread.sleep(10000)
```
Comment (Ph0tonic) [inline app/src/androidTest/java/ch/epfl/sdp/MapActivityTest.kt:85]: What is the reason to wait so long ?

File: app/src/main/java/ch/epfl/sdp/MapActivity.kt [hunk 2]
```diff
@@ -89,6 +89,11 @@ class MapActivity : MapViewBaseActivity(), OnMapReadyCallback {
             clearWaypoints()
         }
 
+        findViewById<Button>(R.id.return_home).setOnClickListener {
+            DroneMission.returnHome()
+        }
```
Comment (Dragoo417) [inline app/src/main/java/ch/epfl/sdp/MapActivity.kt:94]: Don't set these callbacks in the code, use tha android xml instead. This makes code coverage easier and the code itself more tidy

File: app/src/androidTest/java/ch/epfl/sdp/MapActivityTest.kt (hunk 1 above)
Comment (AlexisTabin) [reply app/src/androidTest/java/ch/epfl/sdp/MapActivityTest.kt:85]: @Ph0tonic I know, I asked the group about that but had no answer so I started with the functionnality that was the easiest to implement. I separated my task into 2 tasks to make it shorter and easier to read, that's all.

Comment (AlexisTabin) [reply app/src/androidTest/java/ch/epfl/sdp/MapActivityTest.kt:85]: And I need to correct this test

**Comment by AlexisTabin** [comment]
This feature is not ready yet, we are waiting for Jonas to adapt a C++ function in Java so we can finish it. 
I'll create a pull request after that

**Pull request closed without merging**